package.json

```diff
@@ -16,6 +16,7 @@
     "lint": "eslint -c .eslintrc.json --no-error-on-unmatched-pattern --ext .js,.jsx decidim-*",
     "markdownlint": "markdownlint '**/*.md'",
     "stylelint": "stylelint decidim-*/**/*.scss",
+    "erblint": "./.github/run_erblint.sh",
```

Comment:
We should not put anything in the `.github` folder to the main `package.json` because as far as I understand, the stuff in the `.github` folder is only related to the GitHub actions.

We should either
a) keep running it through the `.github/run_erblint.sh` script in the actions
OR
b) create a ruby script to do the same at the root `bin` folder

Reply:
I lean towards b), an erb linter is not a .github thing

Reply:
Also to note that if we have a ruby script, we should generally avoid running it through the `package.json` scripts.

The `package.json` should be reserved for NPM scripts. E.g. we moved `markdownlint` there when we migrated from the Ruby based `mdl` to the NPM based `markdownlint` (at #9446).

Reply:
I leave the things as they were. I have no enough knowledge about ruby scripting to move it inside `bin` folder :disappointed: